### ISSUE TITLE
Apparent build failure on v1.2.4 on Windows

### ISSUE THREAD

**Issue by upachler** [jiixyj/libebur128#99]
Hi all, 

I'm currently working on a Rust binding for libebur128 (see #98 ). The way the library is built is to download the sources from github, and compile them on the target machine, before compiling the actual Rust binding. 
The build system allows to check out the master as well as tagged versions of libebur128. I'd like to stick to the latest stable version of libebur128, however, I'm finding that v1.2.4 does not compile on Windows. CMake reports an error there. Here's a link to [Travis-CI of a failing build](https://travis-ci.org/upachler/rust-ebur128/jobs/603218881). I assume the relevant error message is:

```
--- stderr
CMake Error at CMakeLists.txt:33 (message):
  queue.h not found, please set ENABLE_INTERNAL_QUEUE_H to ON
```

As a workaround, I'm currently building the master branch, but that's not really a good solution... (haven't tested earlier versions, using them isn't desirable either).

Cheers,

Uwe

**Comment by jiixyj** [comment]
Thanks for the report! Are you able to add your own command line arguments to the CMake invocation? If you add the argument `-DENABLE_INTERNAL_QUEUE_H=ON` CMake should build the library under Windows.

**Comment by upachler** [comment]
Hi Jan,

Not really sure, actually, because I run this via Rust's CMake wrapper. So if there's a new release coming containing the fix in the next couple of weeks, I'd rather wait for that. Is there?

Cheers,
Uwe

**Comment by jiixyj** [comment]
I don't know much about Rust/CMake integration, but maybe this will work: https://docs.rs/cmake/0.1.35/cmake/struct.Config.html#method.define

I agree that a new release is a bit overdue. I'd like to fix #85 first.

**Comment by jiixyj** [comment]
The new release 1.2.5 automatically detects if `<sys/queue.h>` is present on the host or not. So with this, building the library on Windows should be a bit easier, as `-DENABLE_INTERNAL_QUEUE_H=ON` is no longer needed.

So I think we can close this issue.
